trace/widgets/axis_settings.py

```diff
@@ -1,15 +1,18 @@
-from qtpy.QtCore import Qt, Slot
-from qtpy.QtWidgets import QWidget, QCheckBox, QComboBox, QVBoxLayout
+from qtpy.QtCore import Qt, Slot, Signal
+from qtpy.QtWidgets import QWidget, QCheckBox, QComboBox, QPushButton, QVBoxLayout
 
 from pydm.display import Display
 from pydm.widgets import PyDMArchiverTimePlot
 from pydm.widgets.baseplot import BasePlotAxisItem
 
 from config import logger
 from widgets import SettingsTitle, SettingsRowItem
+from widgets.curve_color_palette_modal import CurveColorPaletteModal
```

Comment:
Are you able to include this import in the file `trace/widgets/__init__.py`? This allows the imports here to be simplified to the line below.

```suggestion
from widgets import SettingsTitle, SettingsRowItem, CurveColorPaletteModal
```